src/Loader/ParameterLoader.php

```diff
@@ -11,7 +11,7 @@
  */
 class ParameterLoader
 {
-    const BATCH_SIZE = 20;
+    private const BATCH_SIZE = 20;
```

Comment:
Are you sure that this shouldn't be public instead ?

Reply:
I think no, this constant purpose is to be used in this class and not be used by others class.